ui/app/helpers/includes.ts

```diff
@@ -0,0 +1,17 @@
+import { helper } from '@ember/component/helper';
+
+interface Haystack {
+  includes(needle: unknown): boolean;
+}
+
+export default helper(([haystack, needle]: [unknown, unknown]): boolean => {
+  if (isHaystack(haystack)) {
+    return haystack.includes(needle);
+  } else {
+    return false;
+  }
+});
+
+function isHaystack(haystack?: unknown): haystack is Haystack {
+  return !!haystack && typeof (haystack as Haystack).includes === 'function';
```

Comment:
reading this code made me laugh 😄 🪡 . Just one suggestion though: if we're going to keep the helper as is, maybe add a comment to explain when to use it?

Reply:
Good point! ember-composable-helpers also has [an `includes` helper](https://github.com/DockYard/ember-composable-helpers#includes) but it *only* works on arrays. Will add some more docs and usage guidance 👍